### PR TITLE
Team11 PM 1.3

### PR DESCRIPTION
**Pull request by kostapsimoulis**

### CODE REVIEW

**Comment by smokhov** [comment]
Here are some review comments for this PR. I am not repeating the comments I made for PR #44 regarding package naming and jar placement.

File: src/api/java/openiss/ws/soap/endpoint/OpenISSSOAPServicePublisher.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package api.java.openiss.ws.soap.endpoint;
+
+import api.java.openiss.ws.soap.service.OpenISSSOAPServiceImpl;
+import javax.xml.ws.Endpoint;
+
+public class OpenISSSOAPServicePublisher {
+
+
+    public static void main(String[] args) {
+        Endpoint.publish("http://localhost:7779/ws/xmlread", new OpenISSSOAPServiceImpl());
```
Comment (smokhov) [inline src/api/java/openiss/ws/soap/endpoint/OpenISSSOAPServicePublisher.java]: (A) please agree on with Team 10 in #44 (who use 9090) and refactor hardcoded 7779 to the same port number and refactor it from its hardcoded state to a `Config.java` that both would use for common settings and properties.

File: src/api/java/openiss/ws/soap/client/OpenISSSOAPClient.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.xml.namespace.QName;
+import javax.xml.ws.Service;
+
+@WebServlet(name = "OpenISSSOAPClient", urlPatterns = {"/OpenISSSOAPClient"})
+public class OpenISSSOAPClient extends HttpServlet {
```
Comment (smokhov) [inline src/api/java/openiss/ws/soap/client/OpenISSSOAPClient.java]: (B) Can we also please have a non-servlet plain Java client as well?

File: src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import javax.jws.soap.SOAPBinding.Style;  
+
+/**
+ * Service Endpoint Interface
+ *
+ * @author zephi
+ */
+
+@WebService  
+@SOAPBinding(style = Style.DOCUMENT)
```
Comment (smokhov) [inline src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java]: (C) why `Style.DOCUMENT`? Inconsistency. Please two teams align either way.

File: src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Service Endpoint Interface
+ *
+ * @author zephi
+ */
+
+@WebService  
+@SOAPBinding(style = Style.DOCUMENT)  
+public interface OpenISSSOAPService {
+    @WebMethod String fetchXML(String url, String reqType);
```
Comment (smokhov) [inline src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java]: (D) `fetchXML()` was not a web method in spec, why is it here? Why is it needed?

File: src/api/java/openiss/ws/soap/service/OpenISSSOAPServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+/**
+ * Service Implementation
+ *
+ * @author zephi
+ */
+@WebService(endpointInterface = "api.java.openiss.ws.soap.service.OpenISSSOAPService")
+public class OpenISSSOAPServiceImpl implements OpenISSSOAPService {
+
+    @Override
+    public String fetchXML(String uri, String reqType) {
```
Comment (smokhov) [inline src/api/java/openiss/ws/soap/service/OpenISSSOAPServiceImpl.java]: (D) please remove all this craft; it does not belong to OpenISS

**Comment by smokhov** [comment]
(E) please clean out all the PM1.{1,2} material that does not belong here.

**Comment by smokhov** [comment]
Initial REVIEW DONE

**Comment by smokhov** [comment]
@laihaotao @jashanj0tsingh -- despite this PR being merged already, please review the files and commits to make sure points (A) to (E) are solved above.

File: src/api/java/openiss/ws/soap/endpoint/OpenISSSOAPServicePublisher.java (hunk 1 above)
Comment (laihaotao) [reply src/api/java/openiss/ws/soap/endpoint/OpenISSSOAPServicePublisher.java]: checked (A) has been resolved

File: src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java (hunk 4 above)
Comment (laihaotao) [reply src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java]: checked (D) has been resolved, remove `fetchXML()`.

File: src/api/java/openiss/ws/soap/service/OpenISSSOAPServiceImpl.java (hunk 5 above)
Comment (laihaotao) [reply src/api/java/openiss/ws/soap/service/OpenISSSOAPServiceImpl.java]: checked, (D) all crafts have been removed.

Comment (laihaotao) [reply src/api/java/openiss/ws/soap/service/OpenISSSOAPServiceImpl.java]: by the way, there are two (D) reviews this one and the one above.

File: src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java (hunk 3 above)
Comment (laihaotao) [reply src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java]: `style` has been changed to `SOAPBinding.Style.RPC`.

File: src/api/java/openiss/ws/soap/client/OpenISSSOAPClient.java (hunk 2 above)
Comment (laihaotao) [reply src/api/java/openiss/ws/soap/client/OpenISSSOAPClient.java]: can't find any solution for this review.

**Pull request merged**